lib/cryptoexchange/exchanges/coinrail/services/market.rb

```diff
@@ -0,0 +1,56 @@
+module Cryptoexchange::Exchanges
+  module Coinrail
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+    
+        def fetch
+          raw_output = super(ticker_url)
+          output = []
+          raw_output.map do |market|
+            if market[0] == "btc_market" || market[0] == "krw_market"
+              market[1].each { |pair|
+                output.push(pair)
+              }
+            end
+          end
+          adapt_all(output)
+        end
+
+        def ticker_url
+          # No API endpoint available to obtain tickers (as at 29/01/2018), hence the URL below:
+          "https://coinrail.co.kr/main/market_info?v=#{(Time.now.to_f * 1000).to_i}"
```

Comment:
great you find this url that state all pairs and ticker, so maybe we can use this to list app pairs and not to use `yml` file

btw is it necessary to add params v ? cause it will make test fail (vcr can not handle url changing)

Reply:
and you can use this url to list pairs like:
grab 
```
"cs_name": "BTC",
"ce_name": "ETH",
```
and it's done!

Reply:
and if you use this url in paris.rb, you can extract it out to `exchanges/coinrail/market.rb` and give it another name, so we can father all url in one place

Reply:
Ah OK, will amend and push the revision up :)

Reply:
@Swingcloud added in commit aee6cdf!